### PR TITLE
Print styling enhancements

### PR DESCRIPTION
**Pull request by tsop14**
Keywords are sometimes really long and labels might be - delimits height of all metadata descriptions to try to keep things on the same page. 

Firefox had different display to Chrome - styling changes to make image appear in both.

### CODE REVIEW

File: kahuna/public/stylesheets/main.css [hunk 1]
```diff
@@ -2131,14 +2131,15 @@ FIXME: what to do with touch devices
...
     }
 
-    .image-details {
+    .image-canvas, .image-details {
         height: auto;
         width: auto;
     }
 
-    .image-info__description {
+    dd {
```
Comment (jamesgorrie) [inline kahuna/public/stylesheets/main.css]: Do these `dd`s have classes?

**Comment by jamesgorrie** [comment]
Could we just hide the keywords - or are they useful?
Otherwise :+1:

File: kahuna/public/stylesheets/main.css (hunk 1 above)
Comment (tsop14) [inline kahuna/public/stylesheets/main.css]: The dd used for keywords does not :disappointed:

Comment (jamesgorrie) [inline kahuna/public/stylesheets/main.css]: Could it? We've avoided this just incase we accidentally target a stray `dd`...

**Comment by tsop14** [comment]
> Could we just hide the keywords - or are they useful?

Would rather keep it, just in cases.

**Pull request merged**